### ISSUE TITLE
PowerSaveBlocker not working on Linux

### ISSUE THREAD

**Issue by KeitIG** [electron/electron#3969]
Hi !

- Electron 0.36.2
- Ubuntu 14.04

I've implemented an option in my app to prevent sleep mode. It's working fine on Windows, but not on Ubuntu. Any ideas ?

Code is pretty simple:
``` javascript
ipcMain.on('toggleSleepBlocker', (event, toggle, mode) => {

    if(toggle) {
        instance.sleepBlockerID = powerSaveBlocker.start(mode);
    } else {
        powerSaveBlocker.stop(instance.sleepBlockerID);
        delete(instance.sleepBlockerID);
    }
});
```

**Comment by zcbenz** [comment]
This is an upstream bug: https://code.google.com/p/chromium/issues/detail?id=246060.

**Comment by xywz** [comment]
Fixed in upstream: https://bugs.chromium.org/p/chromium/issues/detail?id=246060#c43.

**Comment by KeitIG** [comment]
Seems to be fixed with Electron 1.2.2 and Chrome 51
